api/metadata_processor_test.go

```diff
@@ -27,6 +28,7 @@ type MetadataProcessorSuite struct {
 }
 
 func (suite *MetadataProcessorSuite) SetupSuite() {
+	log.SetLevel(log.DebugLevel)
```

Comment:
please don't commit this. Not everywhere we need debug level.

Reply:
removed